pkgs/os-specific/linux/nvidia-x11/persistenced.nix

```
@@ -27,6 +32,9 @@ stdenv.mkDerivation rec {
       $out/bin/nvidia-persistenced
   '';
 
+  NIX_CFLAGS_COMPILE = [ "-I${libtirpc.dev}/include/tirpc" ];
+  NIX_LDFLAGS = [ "-L${libtirpc.dev}/lib" "-ltirpc" ];
```

this happens by default
```suggestion
  NIX_LDFLAGS = [ "-ltirpc" ];
```

I guess the others are still needed.